flamingpy/utils/viz.py

```diff
@@ -250,6 +270,7 @@ def draw_EGraph(
         "weight": "Weights",
         "index": "Indices",
     }
+
     name = title_dict.get(label, label)
     if title and label:
         ax.set_title(name)
```

Comment:
I am not a big fan of this title dictionary. Why not put in a string in directly? Or offer both options? i.e. if title is a boolean, use this method and else use the string directly

Reply:
I resolved this by putting it in another function that is called from both the matplotlib and plotly draw methods